### PR TITLE
refactor cljs actions a bit

### PR DESCRIPTION
**Pull request by opqdonut**

### CODE REVIEW

File: src/cljs/rems/actions/approve_reject.cljs [hunk 1]
```diff
@@ -18,33 +16,21 @@
...
-            {:params {:application-id application-id
-                      :comment comment}
-             :handler (flash-message/default-success-handler
-                       :actions
-                       description
-                       (fn [_]
-                         (collapse-action-form action-form-id)
-                         (on-finished)))
-             :error-handler (flash-message/default-error-handler :actions description)}))
+   (command! "approve" {:application-id application-id
```
Comment (opqdonut) [inline src/cljs/rems/actions/approve_reject.cljs]: should the command name be a keyword? should we reuse the command names like `:application.command/approve`?

File: src/cljs/rems/actions/approve_reject.cljs [hunk 2]
```diff
@@ -18,33 +16,21 @@
...
+                        :comment comment}
+             {:description [text :t.actions/approve]
+              :collapse action-form-id
```
Comment (opqdonut) [inline src/cljs/rems/actions/approve_reject.cljs]: should we just compose the collapse into on-finished here?

File: src/cljs/rems/actions/approve_reject.cljs (hunk 1 above)
Comment (luontola) [reply src/cljs/rems/actions/approve_reject.cljs]: We could also have it under a `:type` key, the same way as in the backend.

Comment (luontola) [reply src/cljs/rems/actions/approve_reject.cljs]: Reusing the command names would make it easier to find every place where the command is used.

File: src/cljs/rems/actions/approve_reject.cljs (hunk 2 above)
Comment (luontola) [reply src/cljs/rems/actions/approve_reject.cljs]: Maybe not. The `command!` is anyways very specific to actions.

**Pull request merged**